### PR TITLE
added fix for navbar tests 

### PR DESCRIPTION
**Pull request by isVivek99**
### What is the change?
 fix tests for navbar (#164)

### Is it bug?
No
- Steps to repro
- Expected
- Actual

### \*Dev Tested?

- [x] Yes
- [ ] No

 ### Tested on
NodeJs environment

### Before / After Change Screenshots
Screenshot 2023-07-05 at 7 15 23 AM

### CODE REVIEW

File: app/components/common/navbar/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  //     navbarPages: [
+  //       {
+  //         icon: 'Home',
+  //         text: 'Home',
+  //         href: '/',
+  //         id: 0,
+  //         visibleOnDesktop: true,
+  //       },
+  //       {
+  //         icon: 'Calendar',
```
Comment (harshith-venkatesh) [inline app/components/common/navbar/index.test.tsx:52]: Are the comments required here @isVivek99

**Review by harshith-venkatesh — CHANGES_REQUESTED** [review]

**Comment by harshith-venkatesh** [comment]
Can you also add before and after changes for test coverage?

File: app/components/common/navbar/index.test.tsx (hunk 1 above)
Comment (isVivek99) [reply app/components/common/navbar/index.test.tsx:52]: we need to fix this test, once we get a fix simulating react-router's working in nodejs environment. That is why i was proposing to move to `vitest`. eg: [here](https://github.com/mcansh/stubbbbbs/blob/main/test/setup-test-env.ts)

**Comment by harshith-venkatesh** [comment]
Thank for improving the test coverage and integration of remix stub

**Review by harshith-venkatesh — APPROVED** [review]

**Pull request merged**